### ISSUE TITLE
can you please have a Ruby 2.4 compatible release soon?

### ISSUE THREAD

**Issue by busterb** [banister/method_source#44]
At the moment we have to specify the gem for this (by way of 'pry') by pointing to master in our project's Gemfile:

gem 'method_source', git: 'https://github.com/banister/method_source', branch: 'master'

I know it has been a while, but could you please make a new release to Rubygems?

**Comment by banister** [comment]
@busterb i just installed ruby 2.4 and method_source seems to work fine with it. Out of interest, what features of 2.4 are not supported by the existing method_source release?

**Comment by banister** [comment]
Ah, just this https://github.com/banister/method_source/pull/39/files  and the related warnings that come up? got it

**Comment by danielpclark** [comment]
method_source works for me on Ruby 2.3 series, but in 2.4.2 I get:

```
/usr/share/rvm/rubies/ruby-2.4.2/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- method_source (LoadError)
	from /usr/share/rvm/rubies/ruby-2.4.2/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /home/danielpclark/dev/faster_path/test/monkeypatches/faster_path_test.rb:2:in `<top (required)>'
	from /usr/share/rvm/rubies/ruby-2.4.2/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /usr/share/rvm/rubies/ruby-2.4.2/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /home/danielpclark/.rvm/gems/ruby-2.4.2@global/gems/rake-12.0.0/lib/rake/rake_test_loader.rb:15:in `block in <main>'
	from /home/danielpclark/.rvm/gems/ruby-2.4.2@global/gems/rake-12.0.0/lib/rake/rake_test_loader.rb:4:in `select'
	from /home/danielpclark/.rvm/gems/ruby-2.4.2@global/gems/rake-12.0.0/lib/rake/rake_test_loader.rb:4:in `<main>'
rake aborted!
Command failed with status (1)
/home/danielpclark/.rvm/gems/ruby-2.4.2@global/gems/rake-12.0.0/exe/rake:27:in `<top (required)>'
```

**Comment by banister** [comment]
@busterb just released 0.9.0 that is 2.4 compat, let me know how it goes for you
